src/content/docs/plugin/deep-linking.mdx

```diff
@@ -211,6 +252,74 @@ pub fn run() {
   </TabItem>
 </Tabs>
 
+## Testing
+
+There is some caveats on testing deep links for your application.
+
+### Desktop
+
+Deep links are only triggered for installed applications on desktop.
+On Linux and Windows you can circumvent this using the [`register_all`] Rust function,
+which registers all configured schemes to trigger the current executable:
+
+```rust title="src-tauri/src/lib.rs"
+use tauri_plugin_deep_link::DeepLinkExt;
+
+#[cfg_attr(mobile, tauri::mobile_entry_point)]
+pub fn run() {
+    tauri::Builder::default()
+        .plugin(tauri_plugin_deep_link::init())
+        .setup(|app| {
+            #[cfg(any(windows, target_os = "linux"))]
+            {
+                use tauri_plugin_deep_link::DeepLinkExt;
+                app.deep_link().register_all()?;
+            }
+            Ok(())
+        })
+        .run(tauri::generate_context!())
+        .expect("error while running tauri application");
+}
+```
+
+:::note
+Installing an AppImage that supports deep links on Linux requires an AppImage launcher to integrate the AppImage with the operating system.
+Using the `register_all` function you can support deep links out of the box, without requiring your users to use external tools.
```

Comment:
i kindaaa wanna mention here that moving the AppImage will invalidate the registered deep link

Reply:
time for a dedicated section

Reply:
pushed